gnocchi/tests/test_aggregates.py

```diff
@@ -483,12 +509,39 @@ def test_aggregated_some_overlap_with_fill_null(self):
              numpy.timedelta64(60000000000, 'ns'), 3.0),
         ], list(output))
 
+        output = processor.aggregated([
+            tsc1['return'], tsc2['return']],
+            operations=["-", ["metric"] + tsc1['return'][0],
+                        ["metric"] + tsc2['return'][0]
+                        ], fill=0)["aggregated"]
```

Comment:
should this be fill='null'? it's exactly same as change above.

Reply:
fixed